frontend/khasi-language-explorer/style.css

```
@@ -0,0 +1,359 @@
+/* style.css for Khasi Language Explorer */
+
+.khasi-main {
+    padding-top: 70px;
+    /* Account for navbar */
+    background: var(--bg-color, #0c1410);
+    color: var(--text-color, #fff);
+    font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)